remote-install.sh

```diff
@@ -5,6 +5,11 @@ source ./sg-server-config.sh
 export SCRIPT_BASE_DIR=$(dirname "$(readlink -f "$0")")
 cd $SCRIPT_BASE_DIR
 
+# this is a hack to allow installation in a situation where there isn't a proper init and packages complain about a lack of a runlevel
```

Comment:
So far the script is very Debian-oriented, not sure how that would work in a container, I guess this hack is to allow for running into a non-Debian container?

Sorry, I know next to nothing about containers :-(

Reply:
Initially I want to get this running in a Debian container exactly as is run in Spamgourmet production in order to allow testing and easy development. I am basing off a Debian based Perl image.

Once I have one thing working on that I can start to expand the availability and port to other systems. Changing the base image is just changing one word at the top of the file. After that there will be lots of errors and so porting to something like Alpine (one of the best container Linux versions) involves an unpredictable amount of work depending on how compatible things turn out to be.

Reply:
FYI the script does not install a real clone of spamgourmet - I replaced apache with lighttpd because I lack experience with both and I find lighttpd easier to configure...

Reply:
@vasile-gh in some sense it's worse than that since there are some secret parts of the main spamgourmet service that have never been published. Right now I don't care and just want to get anything running which could run a public service that developers could experiment on.